webpack/scenes/ContentViews/Details/ContentViewDetailActions.js

```diff
@@ -161,6 +164,20 @@ export const getContentViewFilters = (cvId, params) => get({
   url: api.getApiUrl('/content_view_filters'),
 });
 
+export const getContentViewAffectedActivationKeys = (cvId, params) => get({
```

Comment:
Is this endpoint used for anything other than "affected activation keys". 
If so perhaps simply 'getActivationKeys' would be apt.